doc/source/whatsnew/v0.22.0.rst

```diff
@@ -67,6 +68,7 @@ pandas 0.20.3 without bottleneck, or pandas 0.21.x), use the ``min_count``
 keyword.
 
 .. ipython:: python
+   :okwarning:
 
    pd.Series([]).sum(min_count=1)
```

Comment:
For the ones in this file you could actually also add `dtype="float64"` ? (because in the end that's what we wanted to show here, how sum on an empty, numeric series works, and specifying the dtype works fine on all versions)

Reply:
It's a whatsnew of an old version, so I would rather not update the code examples itself. In that version no FutureWarning will be raised to the user